src/main/java/com/faforever/client/chat/ChatUserService.java

```diff
@@ -33,6 +46,8 @@
 @RequiredArgsConstructor
 public class ChatUserService implements InitializingBean {
 
+  private static final ClassPathResource EMOTICONS_JSON_FILE_RESOURCE = new ClassPathResource("images/emoticons/emoticons.json");
```

Comment:
It would be better to create an EmoticonService rather than embed it in this one as they don't have much to do with each other